networkquality: add page (#7444)
tldr-pages/tldr: pages/osx/networkquality.md

# networkQuality

> > Measure the network quality by connecting to the Internet.

Reviewer: I think we don't have a better link than this right now.
```suggestion
> Measure the network quality by connecting to the internet.
> More information: <https://support.apple.com/HT212313>.
```

Author: Yep, I tried searching for the page -- could not find anything :(

> - Test the network quality for the default interface:

Reviewer: Do this command run a speed test always? :thinking: I think the "quality" indicator is the `Responsiveness` (RPM) indicator.

Author: I think in my opinion it always runs a speed test.

`networkQuality`

- Test sequential upload and download instead of parallel:

`networkQuality -s`

- Test a specified network interface:

`networkQuality -I {{en0}}`

> - Display a verbose test output:

Reviewer: ```suggestion
-  Test the network quality with verbose output:
```

# networkQuality

> Measure the network quality by connecting to the internet.
> More information: <https://support.apple.com/HT212313>.

- Test the network quality for the default interface:

`networkQuality`

> - Test sequential upload and download instead of parallel:

Reviewer 2: ```suggestion
- Tes tthe network quality, by sequentially instead of parallel uploading and downloading:
```

Reviewer 3: Typo, `Tes tthe` → `Test the` @marchersimon?
